ansible/roles/loadbalancer/templates/haproxy/haproxy_run.sh.j2

```diff
@@ -1,17 +1,5 @@
 #!/bin/bash -x
 
-{% if enable_letsencrypt | bool %}
-# Copy LetsEncrypt-managed certificates to HAProxy cert folder
-le_base=/etc/letsencrypt/live
-if [[ -d "$le_base" ]]; then
-    domains=$(find $le_base -mindepth 1 -type d -exec basename {} \;)
-    for domain in $domains; do
-        cat "$le_base/$domain/fullchain.pem" "$le_base/$domain/privkey.pem" \
-            >"/etc/haproxy/certs.d/$domain.pem"
-    done
-fi
-{% endif %}
```

Comment:
Do we still need this? What happens when haproxy starts up?

Reply:
The full cert is loaded in a volume, and it is no longer HAProxy's responsibility to craft haproxy.pem from the pieces created by letsencrypt.

Reply:
👍